indico/web/forms/fields.py

```diff
@@ -255,7 +255,7 @@ class CategoryField(HiddenField):
     widget = JinjaWidget('forms/category_picker_widget.html')
 
     def __init__(self, *args, **kwargs):
-        self.select_leaf_only = kwargs.pop('select_leaf_only', False)
+        self.allow_action_on = kwargs.pop('allow_action_on', {})
```

Comment:
This is an empty dict, not an empty set.

And I'd rather have separate kwargs ... Right now you don't even know what you are supposed to pass until you look at the template.